### PR TITLE
Pm/display version from GitHub

### PR DESCRIPTION
**Pull request by PatrikMatiasko**

### CODE REVIEW

File: http-gateway/web/public/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8" />
   <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
   <link href="https://fonts.googleapis.com/css?family=Mulish:400,400i,700" rel="stylesheet">
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (jkralik) [inline http-gateway/web/public/index.html:8]: Do we want to have fixed size to 768 ?

File: http-gateway/web/src/containers/App/AppLayout/AppLayout.tsx [hunk 2]
```diff
@@ -79,7 +122,25 @@ const AppLayout: FC<Props> = (props) => {
...
+                        appStore.version.latest && (
+                            <VersionMark
+                                severity={versionMarkData.severity}
+                                update={
+                                    versionMarkData.severity !== severities.SUCCESS
+                                        ? {
+                                              text: _(t.clickHere),
+                                              onClick: (e) => {
+                                                  e.preventDefault()
+                                                  window.open(GITHUB_VERSION_URL, '_blank')
```
Comment (jkralik) [inline http-gateway/web/src/containers/App/AppLayout/AppLayout.tsx]: use `buildInformation.releaseUrl` it it set - github URL is not pretty. If empty dont put the link in widget.

File: http-gateway/web/src/containers/App/AppInner/AppInner.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ const getBuildInformation = (wellKnownConfig: WellKnownConfigType) => ({
     commitHash: wellKnownConfig?.commitHash || '',
     commitDate: wellKnownConfig?.commitDate || '',
     releaseUrl: wellKnownConfig?.releaseUrl || '',
-    version: wellKnownConfig?.version || '',
+    version: wellKnownConfig?.version || '0.0.0',
```
Comment (jkralik) [inline http-gateway/web/src/containers/App/AppInner/AppInner.tsx]: This informations are in hub under object `buildInfo`:
image

Comment (jkralik) [reply http-gateway/web/src/containers/App/AppInner/AppInner.tsx]: I want to put them the similar way also in client-application.

File: http-gateway/web/public/index.html (hunk 1 above)
Comment (PatrikMatiasko) [reply http-gateway/web/public/index.html:8]: Ondrej wants app without scrollbars, so I removed this meta tag.

File: http-gateway/web/src/containers/App/AppLayout/AppLayout.tsx (hunk 2 above)
Comment (PatrikMatiasko) [reply http-gateway/web/src/containers/App/AppLayout/AppLayout.tsx]: buildInformation.releaseUrl return url for current version but we want to show latest version.

File: http-gateway/web/src/containers/App/AppInner/AppInner.tsx (hunk 3 above)
Comment (PatrikMatiasko) [reply http-gateway/web/src/containers/App/AppInner/AppInner.tsx]: So I can use it from buildInfo also in client app ?

Comment (jkralik) [reply http-gateway/web/src/containers/App/AppInner/AppInner.tsx]: Yes I will move all these properties under `buildInfo` object:
```
    buildDate: wellKnownConfig?.buildInfo?.buildDate || '',
    commitHash: wellKnownConfig?.buildInfo?.commitHash || '',
    commitDate: wellKnownConfig?.buildInfo?.commitDate || '',
    releaseUrl: wellKnownConfig?.buildInfo?.releaseUrl || '',
    version: wellKnownConfig?.buildInfo?.version || '0.0.0',
```

File: http-gateway/web/src/containers/App/AppLayout/AppLayout.tsx (hunk 2 above)
Comment (jkralik) [reply http-gateway/web/src/containers/App/AppLayout/AppLayout.tsx]: You are right - you need to read json from GITHUB_VERSION_URL to get property "html_url":
```json
"html_url": "https://github.com/plgd-dev/hub/releases/tag/v2.7.19",
```

File: http-gateway/web/src/constants.ts [hunk 4]
```diff
@@ -5,3 +5,5 @@ export const shadowSynchronizationStates = {
     ENABLED: 'ENABLED',
     DISABLED: 'DISABLED',
 }
+
+export const GITHUB_VERSION_URL = 'https://api.github.com/repos/plgd-dev/client-application/releases/latest'
```
Comment (jkralik) [inline http-gateway/web/src/constants.ts]: https://api.github.com/repos/plgd-dev/hub/releases/latest

File: http-gateway/web/src/containers/App/AppLayout/AppLayout.tsx (hunk 2 above)
Comment (PatrikMatiasko) [reply http-gateway/web/src/containers/App/AppLayout/AppLayout.tsx]: So, you wanna show github release url, no that json ;)

**Comment by jkralik** [comment]
One question regarding - why it is yellow ?
image

In this case I expect green .
`v2.7.19-pr` vs  `v2.7.19`

**Review by jkralik — APPROVED** [review]

**Pull request merged**